backend/app/utils.py

```
@@ -669,23 +660,12 @@ def download_audio_bytes(url: str) -> tuple[bytes | None, str | None]:
         return None, f"Failed to download audio from URL: {str(e)}"
 
 
-def resolve_audio_url(url: str, mime_type: str) -> tuple[str, str | None]:
-    """Download audio from a public URL and write to temp file. Returns (file_path, error)."""
+def resolve_audio_url(url: str, mime_type: str) -> tuple["AudioRef | None", str | None]:
```

Ruff (0.15.15)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)